src/Chapter3.hs

```diff
@@ -344,6 +344,20 @@ of a book, but you are not limited only by the book properties we described.
 Create your own book type of your dreams!
 -}
 
+data Author = Author 
+  { authorName :: String
+  } deriving (Show)
+
+data ISBN = ISBN 
+  { num :: String
+  } deriving (Show)
```

Comment:
Nice idea about creating custom types for different fields! Later in the chapter we talk about newtypes, and you can see that `newtypes` are perfect for this scenario :100: